buildprocess/configureWebpack.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var StringReplacePlugin = require("string-replace-webpack-plugin");
 
 // If node-sass starts hanging, uncomment this line:
-//process.env.UV_THREADPOOL_SIZE = 128;
+process.env.UV_THREADPOOL_SIZE = 128;
```

Comment:
Having this line uncommented seems to make my Windows build hang, or at least take a really really long time.